Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -45,6 +82,36 @@ public TValue Get<TValue>()
         }
 
         ////TODO: proper message if value type isn't right
+        /// <summary>
+        /// Check if the action button is pressed
+        /// </summary>
+        /// <returns>True if the button is activated over the button threshold. False otherwise</returns>
+        /// <remarks>
+        /// The following example shows how to read a value from a <see cref="PlayerInput"/> message.
+        ///
+        /// <example>
+        /// <code>
+        /// [RequireComponent(typeof(PlayerInput))]
```

Comment:
Does this work without using statements?

Reply:
I matched other examples in the PlayerInput.cs file

Reply:
This is fixed